elpy.el

```diff
@@ -1335,6 +1335,23 @@ for."
 ;;;;;;;;;;;;;;;;;;;;;;
 ;;; Find Project Files
 
+(defcustom elpy-ffip-prune-patterns
+  (let ((extras ()))
+    (dolist (dir elpy-project-ignored-directories)
+      (let ((pat (concat "*/" dir "/*")))
+        (unless (member pat ffip-prune-patterns)
+          (push pat extras))))
+    (dolist (ext completion-ignored-extensions)
+      (unless (string-match "/$" ext)
+        (let ((pat (concat "*" ext)))
+          (unless (member pat ffip-prune-patterns)
+            (push pat extras)))))
+    (append extras ffip-prune-patterns))
```

Comment:
Initializing one defcustom on the basis of another leads to user confusion. It's a good idea to have a function that merges the two defcustoms instead. Also, more readable.